app/src/main/java/com/austindroids/austinfeedsme/AustinFeedsMeApplication.java

```diff
@@ -1,23 +1,20 @@
 package com.austindroids.austinfeedsme;
 
-import android.support.multidex.MultiDexApplication;
-
-import com.austindroids.austinfeedsme.components.ApplicationComponent;
-import com.austindroids.austinfeedsme.components.DaggerApplicationComponent;
-import com.austindroids.austinfeedsme.modules.ApplicationModule;
-import com.austindroids.austinfeedsme.modules.DataModule;
+import com.austindroids.austinfeedsme.di.components.DaggerAppComponent;
+import com.austindroids.austinfeedsme.di.modules.DataModule;
 import com.crashlytics.android.Crashlytics;
 import com.facebook.stetho.Stetho;
 import com.frogermcs.androiddevmetrics.AndroidDevMetrics;
 import com.google.firebase.database.FirebaseDatabase;
 
+import dagger.android.AndroidInjector;
+import dagger.android.support.DaggerApplication;
 import io.fabric.sdk.android.Fabric;
 
 /**
  * Created by darrankelinske on 4/7/16.
  */
-public class AustinFeedsMeApplication extends MultiDexApplication {
-    private ApplicationComponent applicationComponent;
+public class AustinFeedsMeApplication extends DaggerApplication {
```

Comment:
If you don't extend `MultiDexApplication` then you must call `MultiDex.install(this)` in `attachBaseContext()` after the super call.

Reply:
Or he can do this : extends MultiDexApplication implements HasActivityInjector. wdyt ?

Reply:
I chose the former. My real goal is to use proguard and not have to use multidex. Thank you!